tests/end-to-end/tests/searchPage.cy.ts

```diff
@@ -0,0 +1,44 @@
+describe('Search Page', ()=>{
+  beforeEach(() => {
+    cy.login();
+  });
+  it('renders correctly in preview mode', ()=> {
+    cy.visit(
+      '/content/typerefinery-showcase/pages/pages/search.html'
```

Comment:
how do we know it has been rendered? is there a screenshot?

Reply:
Will try to implement the screenshot.